terraform/install.sh

```diff
@@ -21,8 +21,39 @@ set -o errexit  # Exit on error
 
 log() { echo "$1" >&2; }
 
-installTerraform()
-{
+getBillingAccount() {
+  log "Checking for billing accounts"
+  found_accounts=$(gcloud beta billing accounts list --format="value(displayName)" --filter open=true)
+  if [ -z "$found_accounts" ] || [[ ${#found_accounts[@]} -eq 0 ]]; then
+    log "No active billing accounts were detected. In order to create a project, Sandbox needs to have at least one billing account"
```

Comment:
Rephrase this: "In order to create sandboxed environment, the script needs to create a new GCP project and associate it with an active billing account"